test/generate_parse_fixture_yml.py

```diff
@@ -17,7 +18,16 @@ def generate_parse_fixture(example):
     path = os.path.join("test", "fixtures", "dialects", dialect, root + ".yml")
     with open(path, "w", newline="\n") as f:
         r = None
+
         if tree:
+
+            # Check we don't have any base types or unparsable sections
+            types = tree.type_set()
```

Comment:
Nice!

Reply:
Yes I've been meaning to add an `unparsable` check for ages. Stops code reviewers having to check for this. Pretty pleased we have this now. And checking for no `base` segments is an added bonus.

Reply:
You shall not parse!!

![](https://i.kym-cdn.com/entries/icons/facebook/000/002/144/You_Shall_Not_Pass!_0-1_screenshot.jpg)